components/dashboard/Header.js

```diff
@@ -15,10 +15,10 @@ export default function Header() {
           </p>
         </div>
       </div>
-      <div className="self-end flex gap-x-4">
-        <Button text='Contribution Guide' href='https://github.com/asyncapi?type=source#-contribute-to-asyncapi' target='_blank'/>
-        <GithubButton className="mt-5 lg:mt-0" />
-        <SlackButton className="mt-5 lg:mt-0" />
+      <div className="self-end flex gap-x-2 mt-3">
```

Comment:
here I added margin top-3 is for better UI. The header and these buttons were very close